src/syngen/ml/metrics/metrics_classes/metrics.py

```diff
@@ -965,21 +975,22 @@ def diversity(statistics):
         else:
             return 0
 
-    def __automated_silhouette(self):
-        silhouette_scores = []
+    @timing
+    def __automated_davies_bouldin(self):
```

Comment:
May be it makes sense to rename a function (and a scores in the next line) irrelevant to used algorithm?